scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -282,7 +309,8 @@ class FossId(
 
                 val url = pkg.vcsProcessed.url
                 val revision = pkg.vcsProcessed.revision.ifEmpty { "HEAD" }
-                val projectCode = convertGitUrlToProjectName(url)
+                val baseProjectCode = convertGitUrlToProjectName(url)
```

Comment:
I find these variable names confusing. What is now called `projectCode`, isn't this actually the `projectName`? And `projectBaseCode` could then be `projectCode` (or `projectUrl`, `projectId` ?).

Reply:
Yes you are right, thinks got messy when I tried to have FossID semantic (scanCode, projectCode) and the concept of "naming convention".
I revamped it and would appreciate your opinion.

Reply:
Yes, the new names are better. Only the `createScan()` function still uses the old ones as parameters names.